docs/ssm-on-aws/ssm-on-aws.md

```diff
@@ -0,0 +1,374 @@
+
+[Adrian Hornsby][AdrianHornsbyMedium] recently wrote an excellent article on [Injecting Chaos to Amazon EC2 using AWS System Manager][InjectingChaosToAmazonEc2]. This highlighted the fact that this was not covered in the ChaosIQ documentation. The article itself does a great job of covering what AWS Systems Manager (SSM) is and how to set it up.
+
+You will need access to an AWS EC2 instance where you can apply some disruption. If you already have an EC2 instance, its worth confirming if it already has an SSM agent included.
+SSM Agent is installed, by default, on Amazon Linux base AMIs dated 2017.09 and later.
+SSM Agent is also installed, by default, on Amazon Linux 2, Ubuntu Server 16.04, and Ubuntu Server 18.04 LTS AMIs.
+If it is not included you can install the [SSM agent manually](https://docs.aws.amazon.com/systems-manager/latest/userguide/sysman-install-ssm-agent.html)
+
+I did not have a readily available EC2 to hand so I created an EC2 Web server with Flask using this [useful guide](https://www.codementor.io/@jqn/deploy-a-flask-app-on-aws-ec2-13hp1ilqy2). This got me an EC2 instance up and running that would respond to an HTTP request.
```

Comment:
Question: do we use 'I' ?

Reply:
No, it should always be in second person active, "you", please. Lesson 1 when I first started writing for O'Reilly. It's more engaging if the reader feels it's about them, and first person reduces that in favour of the writer's perspective only. First person has its place, just not here.